packages/amplify-graphql-transformer-core/src/cdk-compat/template-asset.ts

```diff
@@ -78,6 +87,14 @@ export class InlineTemplate implements InlineMappingTemplateProvider {
   bind(): string {
     return this.content;
   }
+
+  /**
+   *  get the resolver inline template content
+   * @returns string
+   */
+  getInlineTemplate(): string {
```

Comment:
we should just expose a hash function rather than the template content (same with S3 mapping class)

Reply:
Anyways the content is private ,  Is it necessary to expose a hash function here ?

Reply:
I'd rather expose a hash because exposing the whole template might encourage someone in the future to do something wonky with the template when they should instead operate on it through the class interface. It also removes the need to assert the type when calling the method

Reply:
I don't think someone will be able to change the content since its private or till they implement a function to access private content in template class. 

I have made the to expose a hash function instead of returning template